doc/nmstatectl.8.in

```diff
@@ -166,6 +166,14 @@ rebooting.
 .IP \fB--timeout\fR=<\fITIMEOUT\fR>
 the user must commit the changes within \fItimeout\fR, or they will be
 automatically rolled back. Default: 60 seconds.
+.IP \fB--override-iface\fR
+If defined, nmstate will not merge interface current state when applying
+desired state:
```

Comment:
IMO, it would be clearer: nmstate will not merge the current network state to the interfaces' desired state.